src/le.py

```diff
@@ -2472,18 +2585,47 @@ def set_datahub_settings(self, value, should_die=True):
                     values[1])
         self.datahub = value
 
+    def validate_pathname(self, args):
+        """
+        For the '--multilog' option where a wildcard can be used in the directory name.
+        Validates the string that is passed to the agent (LOG-7549).
+        :param args: the pathname given to the agent
+        :return:    True if okay, the agent will die otherwise
+        """
+        pname_slice = args[1:]
+        # validate that a pathname is detected in parameters to agent
+        if len(pname_slice) == 0:
+            die("\nError: No pathname detected - Specify the path to the file to be followed\n" + MULTILOG_USAGE, EXIT_OK)
```

Comment:
If it is a validation, shouldn't we return False and leave any killing action to the caller?